Cargo.toml

```diff
@@ -5,12 +5,14 @@ authors = ["Gedalia Kott <gedkott@gmail.com>"]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-[lib]
-name = "bencoding"
-path = "src/lib.rs"
 
 [[bin]]
 name = "bit_torrent"
 path = "src/main.rs"
 
 [dependencies]
+reqwest = { version = "0.10", features = ["blocking", "json"] }
+sha1 = "0.6.0"
```

Comment:
I need to be able to sha1 some contents of the metainfo file and other content for tracker API requests.

Reply:
some "info" key in the metainfo file tree.